### ISSUE TITLE
Exercise Forms (UI and Saving Data)

### ISSUE THREAD

**Issue by uladkasach** [personalcancertoolkit/openmrs-module-healthybehaviors#14]
https://github.com/personalcancertoolkit/openmrs-module-patientportaltoolkit/issues/257
#3 

Base the forms off of physical copy given. Modify questions as needed.

**Comment by maurya** [comment]
Make a decision on Forms by OpenMRS and Custom

**Comment by uladkasach** [comment]
After long deliberation and more than a day spent attempting to embed htmlformentrys into our application, its clear that we should implement a custom solution. 

- HtmlFormEntery was not built to be embedded into other modules
- HtmlFormEntery is overkill for the type of observation recording we wish to do
- HtmlFormEntery is restricting, here are several examples
    - requires xml to be rendered and grabbed by the server side (disabling the ability to build html w/ js)
    - requires page to reload upon submission
    - requires mixing of client side and server side (form must be generated through server side)
 - HtmlFormEntery has bugs, atleast on my development machine, that disable it from working
- HtmlFormEntery is very complicated - more complicated than it needs to be for a developer
    - this is because it was made as an easy way for non-developers to generate forms (e.g., clinicians)

In summary, HtmlFormEntery was not built for developers and was not built for being embedded into other applications. It will take more work than is reasonable to implement in our application and even if implemented will embed poor web architecture into the module and bring with it an overkill of functionality. Because of this, we will use a custom solution.

**Comment by uladkasach** [comment]
The following commit referenced this issue but github did not pick it up. : https://github.com/personalcancertoolkit/openmrs-module-healthybehaviors/commit/662647c0c62406038e01edc9de91cbf10818a35a, "generate a 'simpleform' for exercise behavior"
